waku/v2/protocol/waku_rln_relay/waku_rln_relay_metrics.nim

```diff
@@ -31,3 +42,48 @@ declarePublicGauge(waku_rln_registration_duration_seconds, "time taken to regist
 declarePublicGauge(waku_rln_instance_creation_duration_seconds, "time taken to create an rln instance")
 declarePublicGauge(waku_rln_membership_insertion_duration_seconds, "time taken to insert a new member into the local merkle tree")
 declarePublicGauge(waku_rln_membership_credentials_import_duration_seconds, "time taken to import membership credentials")
+
+template parseAndAccumulate(collector: Collector, cumulativeValue: float64): float64 =
```

Comment:
@jm-clius , do you think this would be useful in the utils directory? It can be used in `wakunode_setup_metrics.nim`: https://github.com/status-im/nwaku/blob/371016b04fc8467f6a4c09564b26d96d1acd0999/waku/v2/node/wakunode2_setup_metrics.nim#L54

Reply:
Yeah, I think this can go to the collector util. :)

Reply:
Addressed in [6915cd0](https://github.com/status-im/nwaku/pull/1262/commits/6915cd09ba7e3ce55c1b6804a97ddedcc2c95648)